### PR TITLE
Lvnod 23 webpack configuration

### PR DESCRIPTION
**Pull request by AlbaMontis**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LVNOD-23)
* [JIRA subtask](https://ssu-jira.softserveinc.com/browse/LVNOD-37)

## Code reviewers

- [ ] @denisbodnar
- [ ] @shyyka
- [ ] @avoytovich

### Second Level Review

- [ ] @lhalam
- [ ] @4asof4ik

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo
## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=90%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by denisbodnar — DISMISSED** [review]

**Review by denisbodnar — DISMISSED** [review]

**Review by shyyka — DISMISSED** [review]

**Review by avoytovich — DISMISSED** [review]

File: app/src/components/Hello/hello.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+$primary-color: #123;
```
Comment (Vovk-Nazar) [inline app/src/components/Hello/hello.scss]: rename Hello.scss

File: package.json [hunk 2]
```diff
@@ -3,7 +3,12 @@
   "version": "1.0.0",
   "description": "Front End",
   "main": "app.js",
-  "scripts": {},
+  "scripts": {
+    "test": "",
+    "build:prod": "NODE_ENV=production webpack --config ./webpack/webpack.config.prod.js -p",
```
Comment (Vovk-Nazar) [inline package.json]: webpack -p

File: server.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const app = express();
+const PORT = 3000;
+const indexPath = __dirname + '/build/index.html';
```
Comment (Vovk-Nazar) [inline server.js]: use path

**Review by shyyka — DISMISSED** [review]

**Review by denisbodnar — DISMISSED** [review]

**Review by avoytovich — DISMISSED** [review]

File: webpack/webpack.config.dev.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+module.exports = {
+  entry: {
+    app: [entryPath]
+  },
+  output: {
+    path: path.resolve('build'),
+    filename: 'bundle.js'
+  },
+  devServer: {
+    contentBase: 'build'
```
Comment (4asof4ik) [inline webpack/webpack.config.dev.js]: add watch and hot as true

File: server.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const app = express();
+const PORT = 3000;
```
Comment (4asof4ik) [inline server.js]: process.env.PORT || 3000;

File: webpack/webpack.config.dev.js [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+  },
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        use: ['babel-loader']
```
Comment (4asof4ik) [inline webpack/webpack.config.dev.js]: why you use array here?

File: webpack/webpack.config.dev.js [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+      },
+      {
+        test: /\.scss$/,
+        use: ExtractTextPlugin.extract({
+          fallback: 'style-loader',
+          use: ['css-loader', 'sass-loader']
+        })
+      }
+    ],
+    loaders: []
```
Comment (4asof4ik) [inline webpack/webpack.config.dev.js]: loader: [] ???

File: webpack/webpack.config.prod.js [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+	use: ['strip-loader']
+};
+
+let cleanBuildplagin = new CleanWebpackPlugin([buildPath], {
+  root: rootPath,
+  verbose: true,
+  dry: false
+});
+
+devConfig.module.loaders.push(stripLoader);
```
Comment (4asof4ik) [inline webpack/webpack.config.prod.js]: maybe module.rules ???

File: webpack/webpack.config.prod.js [hunk 9]
```diff
@@ -1,23 +1,17 @@
...
 
-let stripLoader = {
-	test: [/\.js$/],
-	exclude: /node_modules/,
-	use: ['strip-loader']
-};
+devConfig.output.path = path.resolve('./build');
 
-let cleanBuildplagin = new CleanWebpackPlugin([buildPath], {
+let cleanBuildplagin = new cleanWebpackPlugin([buildPath], {
```
Comment (denisbodnar) [inline webpack/webpack.config.prod.js]: let cleanBuildPlugin, u, not a, capital P

**Review by denisbodnar — DISMISSED** [review]

**Review by avoytovich — DISMISSED** [review]

**Review by shyyka — DISMISSED** [review]

**Review by denisbodnar — APPROVED** [review]

**Review by avoytovich — APPROVED** [review]

**Review by shyyka — APPROVED** [review]

**Review by 4asof4ik — APPROVED** [review]

**Pull request merged**